pkgs/applications/misc/onboard/default.nix

```diff
@@ -38,6 +40,10 @@ in python3.pkgs.buildPythonApplication rec {
   };
 
   patches = [
+    (substituteAll {
+      src = ./fix-paths.patch;
```

Comment:
According to the eval status `fix-paths.patch` isn't in tree (file doesn't exist).